README.md

```diff
@@ -8,3 +8,15 @@
 A simple calculator application built using the Fyne toolkit.
 
 ![](img/calc-dark.png) &nbsp; ![](img/calc-light.png)
+
+# Installation instructions
+Install Fyne:
+```go get fyne.io/fyne/v2/cmd/fyne```
+
+Install resources for this application:
```

Comment:
I'm not sure what "resources" are - would it not be simpler to say "Install the application by executing:"

Reply:
Well, i m compiling the application, not installing it. The compile gave me problems because of lacking resources.